pipeline-runner/tests/testthat/test-gem2s-2-load_user_files.R

```diff
@@ -443,3 +443,65 @@ test_that("parse_rhapsody_matrix uses RSEC if DBEC corrected counts are missing"
 
   expect_equal(values, expected_values)
 })
+
+
+test_that("read_10x_files removes rows with empty feature names both in count matrix and annotation if present", {
+  counts <- mock_counts()
+  rownames(counts)[1] <- ""
+
+  features <- data.frame(
+    ensid = paste0("ENSFAKE", seq_len(nrow(counts))),
+    symbol = row.names(counts)
+  )
+  features[1,1] = ""
+
+  outdir <- tempdir()
+  sample <- "sample_a"
+  sample_dir <- file.path(outdir, sample)
+  dir.create(sample_dir)
+
+  mock_cellranger_files(counts, features, sample_dir)
+
+  prev_out <- list(config = list(samples = sample, input = list(type = "10x")))
+
+  out <- pipeline::load_user_files(NULL, NULL, prev_out, outdir)$output
```

Comment:
This is not necessary, it's an exported function in the pipeline

```suggestion
  out <- load_user_files(NULL, NULL, prev_out, outdir)$output
```